_includes/use-cases/use-case-consumption-execute-an-identityattributequery.md

```diff
@@ -0,0 +1,18 @@
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case is intended to query the [identity attributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute)
```

Comment:
This use-case is intended to execute an incoming IdentityAttributeQuery (e.g. of a ReadAttributeRequestItem) which returns a list of matching [Identity Attributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute).